From: "TJ" <ubuntu@tjworld.net>
Date: Tue, 21 Nov 2017 21:34:41 +0000
Subject: Don't strip symbols in upstream's Makefile,
 let dh_strip take care of stripping, which allows for debug builds.

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 5ffbd48..ce22c09 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ BIN_DIR = ${DESTDIR}/usr/bin
 all: dmg2img vfdecrypt 
 
 dmg2img: dmg2img.c dmg2img.h mntcmd.h gpt.h dmg2img.o base64.o adc.o 
-	$(CC) -s -o dmg2img dmg2img.o base64.o adc.o -L. -lz -lbz2
+	$(CC) -o dmg2img dmg2img.o base64.o adc.o -L. -lz -lbz2
 
 dmg2img.o: dmg2img.c dmg2img.h
 	$(CC) $(CFLAGS) -c dmg2img.c
@@ -17,11 +17,11 @@ adc.o: adc.c adc.h
 	$(CC) $(CFLAGS) -c adc.c
 
 vfdecrypt: vfdecrypt.c
-	$(CC) $(CFLAGS) -s -o vfdecrypt vfdecrypt.c -lcrypto
+	$(CC) $(CFLAGS) -o vfdecrypt vfdecrypt.c -lcrypto
 
 install: dmg2img vfdecrypt
 	mkdir -p ${BIN_DIR}
-	install -c -s -m 755 -o root -g root dmg2img vfdecrypt $(BIN_DIR)
+	install -c -m 755 -o root -g root dmg2img vfdecrypt $(BIN_DIR)
 
 clean:
 	rm -f dmg2img vfdecrypt *~ *.o core
